tests/tests.py

```diff
@@ -1,3 +1,7 @@
+import sys
+# caution: path[0] is reserved for script path (or '' in REPL)
+sys.path.insert(1, '../similaritymeasures')
+
```

Comment:
These lines are not needed when you install the package. Please remove these 4 lines.

When you do this kind of package development, you should install the library from the source code. I normally clone the library, and then install the local files with pip. After making changes to the source files you then re-install them.

A lot of people find this constant installation annoying.

A good alternative is to use `pip -e` which will make a symbolic link from your site packages folder to the local source code. Essentially then the library's source code becomes editable, and you don't

So with all that said, you can avoid setting paths by simply doing
```bash
git clone https://github.com/cjekel/similarity_measures.git
python -m pip install -e ./similarity_measures
```
which will then let you make changes to the source code and always import the latest code.

Reply:
Thank you for the explanation on how to reinstall everything!

I just removed these lines in my latest commit